flytekit/extend/backend/base_agent.py

```diff
@@ -145,7 +145,7 @@ def metadata_type(self) -> ResourceMeta:
         return self._metadata_type
 
     @abstractmethod
-    def create(self, task_template: TaskTemplate, inputs: Optional[LiteralMap], **kwargs) -> ResourceMeta:
+    def create(self, task_template: TaskTemplate, output_prefix: str, inputs: Optional[LiteralMap], **kwargs) -> ResourceMeta:
```

Comment:
Could we add the output_prefix only to the base class? since other agents in the flytekit don't use it